src/main/daml/Daml/Finance/Account/Util.daml

```diff
@@ -0,0 +1,37 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Account.Util where
+
+import DA.Set qualified as S (isSubsetOf, member)
+import Daml.Finance.Holding.Util (reapplyLocks, releaseAll)
+import Daml.Finance.Interface.Account.Account qualified as Account (Credit(..), Debit(..), I, Transfer(..), exerciseInterfaceByKey)
+import Daml.Finance.Interface.Account.Util (getOwner)
+import Daml.Finance.Interface.Holding.Base qualified as Base (I)
+import Daml.Finance.Interface.Holding.Lockable (getLockers)
+import Daml.Finance.Interface.Holding.Lockable qualified as Lockable (I)
+import Daml.Finance.Interface.Holding.Transferable qualified as Transferable (I)
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (qty)
+import Daml.Finance.Interface.Types.Common (AccountKey(..))
+
+transferImpl : AccountKey -> Account.Transfer -> Update (ContractId Transferable.I)
```

Comment:
Could we add a function description?

Reply:
-- | Default implementation of `transfer` from the `Account` interface.